core/modes/code_languages.py

```diff
@@ -19,6 +19,7 @@ def __init__(self, id: str, spoken_form: str | list[str], extensions: list[str])
     # Language("bash", "bash", ["sh", "bashbook"]),
     Language("batch", "batch", ["bat"]),
     Language("c", "see", ["c", "h"]),
+    Language("community_snippets", "community snippets", ["snippet"]),
```

Comment:
I would just call the id `snippets`. That's what I'm doing in vscode for the syntax highlighter and formatter.